definitions/Service.ts

```diff
@@ -7,7 +7,9 @@
 import * as t from "io-ts";
 
 // required attributes
-const ServiceR = t.interface({});
+const ServiceR = t.interface({
+  scope: t.string
```

Comment:
```suggestion
  scope: t.union([t.literal("LOCAL"), t.literal("NATIONAL")])
```
if you declare scope as a string, when you edit services.xml and you make a typo or you insert a wrong value , the encoding will fail, otherwise the app will get data that it can't handle

Reply:
@Undermaken you are right, but this code gets auto generated (see header)

@chiadav the solution is to use `x-extensible-enum` (see an example [here](https://github.com/teamdigitale/io-functions-commons/blob/master/openapi/definitions.yaml#L37))

Reply:
Perfect, using the `x-extensible-enum` we have the enum and the value validation.